jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/HandlerRegistry.java

```diff
@@ -0,0 +1,56 @@
+package io.opentelemetry.contrib.jfr.metrics;
+
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.contrib.jfr.metrics.container.ContainerConfigurationHandler;
+import io.opentelemetry.contrib.jfr.metrics.cpu.LongLockHandler;
+import io.opentelemetry.contrib.jfr.metrics.memory.ObjectAllocationOutsideTLABHandler;
+import io.opentelemetry.contrib.jfr.metrics.network.NetworkReadHandler;
+import io.opentelemetry.contrib.jfr.metrics.network.NetworkWriteHandler;
+import io.opentelemetry.contrib.jfr.metrics.cpu.ContextSwitchRateHandler;
+import io.opentelemetry.contrib.jfr.metrics.cpu.OverallCPULoadHandler;
+import io.opentelemetry.contrib.jfr.metrics.memory.G1GarbageCollectionHandler;
+import io.opentelemetry.contrib.jfr.metrics.memory.GCHeapSummaryHandler;
+import io.opentelemetry.contrib.jfr.metrics.memory.ObjectAllocationInNewTLABHandler;
+
+
+import java.util.*;
+import java.util.stream.Stream;
+
+public class HandlerRegistry {
+  private static final String SCHEMA_URL = "https://opentelemetry.io/schemas/1.6.1";
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.contrib.jfr";
+  private static final String INSTRUMENTATION_VERSION = "1.7.0-SNAPSHOT";
```

Comment:
Nit:  Maybe open a bug about pulling this version from a generated resources file in the build.